webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -252,6 +252,14 @@
 
                             {# Crash circumstances #}
 
+                            {% if raw.StartupCrash %}
+                            <tr title="{{ fields_desc['raw_crash.StartupCrash'] }}">
+                                <th scope="row">Startup Crash</th>
+                                <td>
+                                    <pre>True</pre>
```

Comment:
This is not going to work. 
You mentioned in IRC that in the raw crash this value might be `'0'`, `'1'` or non-existant. If it's `'0'` the line `{% if raw.StartupCrash %}` will be true. 

I think a better approach would be to handle these values differently. 

The raw crash JSON will probably always be a string (that is a number 0 or 1) and not a fully fledged JSON boolean. 
So, the line `{% if raw.StartupCrash %}` will be true every time the *key* is present in the raw crash. 

Then, when it comes to displaying the value we could have a jinja helper function do a string conversion to something like this:

```python
def display_booleanish(value):
if str(value).lower() in ('1', 'true', 'yes'):
    return "true"
else:
    return "false"
```